learning_resources/fixtures/departments.json

```diff
@@ -271,14 +271,6 @@
       "school_id": null
     }
   },
-  {
-    "model": "learning_resources.LearningResourceDepartment",
-    "fields": {
-      "department_id": "RES",
-      "name": "Supplemental Resources",
-      "school_id": null
-    }
-  },
```

Comment:
I think it would be better now to add a new migration in `data_fixtures/migrations` to remove the RES department (and, eventually, to add the SP department) instead of running the `update_schools_departments` command

Reply:
Done! It worries me that we need to update this in three different places though.